test/conformance/chainsaw/cli/two-results-with-namespace/kyverno-test.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: cli.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: kyverno-test.yaml
+policies:
+- policy.yaml
+resources:
+- resources.yaml
+results:
+- kind: Pod
+  policy: default/test-jmespath
+  resources:
+  - test-invalid
+  result: fail
+  rule: test-jmespath
+- kind: Pod
+  policy: test-jmespath
+  resources:
+  - test-valid1
+  - test-valid2
+  - test-valid3
+  result: pass
+  rule: test-jmespath
+- kind: Namespace
+  policy: namespacevalidation
```

Comment:
Same here. There is no ClusterPolicy called this name anywhere. So there are multiple errors that should be returned. Can all errors be returned at the same time?